lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -560,6 +562,27 @@ def install_a_puppet_msi_on(host, opts)
           end
         end
 
+        # Installs Puppet and dependencies from FreeBSD ports
+        #
+        # @param [Host] host The host to install packages on
+        # @param [Hash{Symbol=>String}] opts An options hash
+        # @option opts [String] :version The version of Puppet to install (shows warning)
+        #
+        # @return nil
+        # @api private
+        def install_puppet_from_freebsd_ports_on( host, opts )
```

Comment:
But none of the other `_on` methods have that ability? This method has the same implementation as `install_puppet_from_deb_on` which gets the host given to it from the `block_on` block on https://github.com/petems/beaker/blob/BKR-326-fix_freebsd_install/lib/beaker/dsl/install_utils/foss_utils.rb#L231

Am I missunderstanding something? I can change the method, but it seems to be the same logic as the rest of the methods.